app/views/patients/show.html.erb

```diff
@@ -126,10 +130,14 @@
           <td><%= cc.last_name %></td>
           <td><%= cc.primary_telephone.present? && cc.primary_telephone.length > 10 ? cc.primary_telephone&.gsub('+1', '')&.gsub(/\D/, '')&.insert(3, '-')&.insert(7, '-') : '' %></td>
           <td><%= cc.email %></td>
-          <td><%= cc.contact_attempts %></td>
+          <td><%= cc.last_date_of_exposure&.to_date&.strftime('%m/%d/%Y') %></td>
+          <td><%= cc.assigned_user %></td>
+          <td style="word-wrap: anywhere;"><%= cc.notes && cc.notes.length > 500 + 3 ? (cc.notes[0..500] + '...') : cc.notes %></td>
```

Comment:
`word-wrap: anywhere` is not supported by IE or Safari. Does `break-word` work well here instead?
https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap#browser_compatibility

Reply:
Yes! I have a working solution. Good catch here!